src/constants.py

```diff
@@ -34,3 +34,8 @@
     'saturday': [5],
     'sunday': [6]
 }
+YELP_LATITUDE = 42.440680
+YELP_LONGITUDE = -76.486043
+YELP_NUMBER_LIMIT = 50  # maximum allowed by yelp
```

Comment:
maximum what allowed by yelp? i don't think you need `number`, `limit` implies it's a number